artemis/modules/nuclei.py

```diff
@@ -480,6 +521,36 @@ def run_multiple(self, tasks: List[Task]) -> None:
                 self._scan(self._workflows, ScanUsing.WORKFLOWS, group_targets, extra_nuclei_args=extra_args)
             )
 
+        # DAST scanning
+        dast_targets: Dict[str, List[str]] = {}
+        dast_targets["other"] = []
+        for keyword, template_data in DAST_SCANNING.items():
+            dast_targets[keyword] = []
+            for task in tasks:
+                param_url = add_common_params_from_wordlist(
+                    get_target_url(task), template_data["params_wordlist"], template_data["param_default_value"]
+                )
+                dast_targets[keyword].append(param_url)
+                # Since there are no params for "other", we can just combine all the params from the different vulns into this
+                dast_targets["other"].append(param_url)
```

Comment:
I'm afraid this wont work - look, there will be 6 times as much URLs for 'other' than for each of the keywords. Can it be made faster, by putting 'other' as a separate, single entry in DAST_SCANNING?